zerver/tests/test_events.py

```diff
@@ -1830,7 +1830,7 @@ def test_change_is_admin(self) -> None:
             ('type', equals('realm_user')),
             ('op', equals('update')),
             ('person', check_dict_only([
-                ('is_admin', check_bool),
+                ('role', check_int_in(UserProfile.ROLE_TYPES)),
                 ('user_id', check_int),
             ])),
         ])
```

Comment:
I would have expected us to have a test for `test_change_is_guest` modified here.  Looks like we just never wrote one; and if we did, it'd fail :).

Reply:
Added c78d7288129ddb555b4a4743a57b15e8e33df391 to address that gap.